### DISCUSSION TITLE
v1.4-alpha.5 - now on TestFlight

### DISCUSSION THREAD

**Discussion by arekf** [yattee/yattee#149] [Announcements]
## What's changed
* Added slide gesture for hiding player (#44) (iOS)
* Added buttons in player controls (#106)
  * Close video
  * Next video in the queue
  * Restart current video
* Added option to video context menu to start playing in PiP (#137) (iOS)
* Added button to close current video in browser controls
* Changed tab navigation to use overlays instead of sheets for channels and playlists (iOS)
* Fixed orientation changes handling (#121) (iOS)
* Fixed action of closing video (#94)
* Removed most of defaults Favorites list (#101)
* Minor controls UI improvements (controls will get soon re-implemented for better UI and performance)
* Other minor fixes and improvements

**Full Changelog**: https://github.com/yattee/yattee/compare/b306819af9c788b13eab86b88c417dcfdeb710b4...v1.4-alpha.5

### [Join TestFlight Beta](https://yattee.stream/beta)

<hr /><em>This discussion was created from the release <a href='https://github.com/yattee/yattee/releases/tag/v1.4-alpha.5'>v1.4-alpha.5 - now on TestFlight </a>.</em>

**Comment by bachig26** [comment]
Refined needed for the player controls and the layout at the moment (alpha5):

1. The **swipe down function to close the fullscreen is available for v1.4.**

2. The controls are messy as you know, and from my use this past few hours of the release, **in fullscreen mode the options that is shown for switching the player and video quality need to in right top and rest of options like "close, PiP, fullscreen, 1x" can be moved left top.**

**_[Portrait]_**
![IMG_6553](https://user-images.githubusercontent.com/96393348/170916436-b3b95fc9-e302-49a7-99db-ae383ff8ee55.jpeg)

**_[Lanscape]_**
![IMG_6554](https://user-images.githubusercontent.com/96393348/170916445-4294b10f-7963-4678-92e9-aeeb895a803e.PNG)

**Comment by bachig26** [comment]
3. The slide gesture for hiding the player is not fluid.

4. when going to fullscreen, sometimes the controls are not fluid as it changes from portrait to landscape or from other side in landscape. And also the rotation to fullscreen and portrait still needs some improvement as it is not proper yet.

**_ps. there comments are based on my experience and you may have already know these issues._**

**Comment by bachig26** [comment]
@arekf the issue with history/ previously played videos being added automatically to the Queue of the currently play video. as the history/ previously played videos plays after the current one ends. This issue needs to be fixed. I thought this was solved, but while using the alpha 5, this one is present.

**Comment by hommegithub** [comment]
Been patiently waiting but it seems that the tvOS 1.4 app won't let my ATV 3 and HD remotes control yattee at all...
I want to switch over to 1.4 but only 1.3 works with the remote, please help!

It goes double playback speed and even gave me a black screen, only leaving only audio playing... There's just nothing showing in terms of controlling the player. 

Also the tv remote on iOS also won't work with yattee 1.4.

**Reply by arekf** [reply]
tvOS version is still the least polished and limited. I only have and tested new remote, I'll try soon to make it usable for the older one or the one in iOS.

**Reply by hommegithub** [reply]
Thanks. I'm also new on GitHub and missed your response. by iOS you meant the ATV remote right?  Please yattee is the only hope without jailbreaking an ATV...

**Reply by arekf** [reply]
I meant older physical remote model and digital remote for ATV available in iOS Control Center.

As a incomplete workaround for now you can use AirPlay menu on your iPhone to control playback in V1.4, see screenshot

![image](https://user-images.githubusercontent.com/367484/175991732-09a978f1-86fd-47bd-b268-99faceb9dbb0.png)

**Comment by bachig26** [comment]
> Been patiently waiting but it seems that the tvOS 1.4 app won't let my ATV 3 and HD remotes control yattee at all...
> 
> I want to switch over to 1.4 but only 1.3 works with the remote, please help!
> 
> 
> 
> It goes double playback speed and even gave me a black screen, only leaving only audio playing... There's just nothing showing in terms of controlling the player. 

Did you tried the latest version from the TestFlight?
Also text in Matrix channels in parallel, so you can have fast answer based on the availability both in GitHub or Matrix.

**Reply by hommegithub** [reply]
Yes that's why I said I've been patiently waiting 😎. I don't see much users on matrix so decided not to sign up and download yet another app...  I'm just surprised why nobody has cared to test ATV remote in the first place. I thought yattee was targeted for tvOS...  There's tons of solutions for iOS already.

**Reply by arekf** [reply]
>  I'm just surprised why nobody has cared to test ATV remote in the first place. I thought yattee was targeted for tvOS... 

The user base of tvOS version is very small. 95% of people use iOS version.

> There's tons of solutions for iOS already.

I don't think so, show me one that isn't a tweak on YouTube app and does not connect to their servers at all.

**Reply by hommegithub** [reply]
> > I'm just surprised why nobody has cared to test ATV remote in the first place. I thought yattee was targeted for tvOS...
> 
> The user base of tvOS version is very small. 95% of people use iOS version.

I'm the unlucky few who still sticks with ATV and Roku which has no workaround lol
Oh man I'm just trying to be a legit Apple fan without yt premium!

> > There's tons of solutions for iOS already.
> 
> I don't think so, show me one that isn't a tweak on YouTube app and does not connect to their servers at all.

Hmm I'd just use the piped web interface for one... It works on most iOS (except my old iPad but yattee won't support iOS 12 either but yea tweaked apps are also alternatives)....

**Comment by hommegithub** [comment]
Thanks for making the tvOS fix a priority!  I'm sorry if this is not the way to report bug for the latest build:
- Older Remotes seem to work with 1.4 (tested on ATV HD with tvOS 16)
- Player control is way too small visually to be useful.
- Invidious not working at all, takes forever to play any streams. 
- captions/subtitles are completely gone, both indivious and piped. 
- first build started crashing on my ATV HD (didn't happen on previous 1.4 builds)

Suggestions:
- both 1.4 and 1.3: options to "play all" for subscriptions, or parse subscription feed onto a playlist.  Or option to add all onto a new playlist (user creates a new list everytime which is not ideal). 
Please maintain v1.3 as this seems to work best with older ATVs and the apple standard player UI 
- any chance to default 1080p quality?
- Subtitles/closed captioning has missing auto-generated English option (not sure if it's a 3rd party YouTube limitation)
- Subtitles/closed captioning options on yattee UI? 
- changing streaming quality no longer works (used to work with adaptive HLS, then I choose 1080p manually)

P.S. Just wanted to stressed that a lightweight solution like Yattee seems to be the last hope for tvOS, I have Kodi but maintaining it is another nightmare.

**Reply by hommegithub** [reply]
Opened a bug with some updates on remote issues
